### PR TITLE
improvement: add toast notification for toggle hotkeys

### LINKED ISSUE #2244: Show toast notification on usage of hotkeys

### PR DESCRIPTION
**Pull request by brandstetterm**
## Description

Add toast notifications to hotkeys where the impact of the hotkey is not directly visible.

Closes #2244 

## Changelist 
- Added translated toast notifications for some hotkeys

### CODE REVIEW

File: public/locales/de/translation.json [hunk 1]
```diff
@@ -178,7 +178,31 @@
     "hotkeysEnabled": "Hotkeys aktiviert!",
     "hotkeysDisabled": "Hotkeys deaktiviert!",
     "hotkeyToggle": "Hotkeys aktivieren",
-    "cheatSheet": "Hotkey Spickzettel"
+    "cheatSheet": "Hotkey Spickzettel",
+    "toggleColumnVisibility": {
+      "hide": "Ausgeblendete Spalten sind nun für dich ausgeblendet",
```
Comment (Resaki1) [inline public/locales/de/translation.json:183]:
```suggestion
      "hide": "Ausgeblendete Spalten sind nun auch für dich ausgeblendet",
```

File: public/locales/de/translation.json [hunk 2]
```diff
@@ -178,7 +178,31 @@
...
+      "show": "Ausgeblendete Spalten sind nun für dich sichtbar"
+    },
+    "toggleShowOtherUsersNotes": {
+      "hide": "Die Kärtchen anderer Teilnehmer sind ausgeblendet",
```
Comment (Resaki1) [inline public/locales/de/translation.json:187]:
```suggestion
      "hide": "Die Kärtchen anderer Teilnehmer sind nun ausgeblendet",
```

File: public/locales/de/translation.json [hunk 3]
```diff
@@ -178,7 +178,31 @@
...
+      "show": "Die Kärtchen anderer Teilnehmer sind nun sichtbar"
+    },
+    "toggleShowAuthors": {
+      "hide": "Die Autoren der Kärtchenn sind ausgeblendet",
```
Comment (Resaki1) [inline public/locales/de/translation.json:191]:
```suggestion
      "hide": "Die Autoren der Kärtchen sind  nun ausgeblendet",
```

File: public/locales/en/translation.json [hunk 4]
```diff
@@ -178,7 +178,31 @@
...
     "hotkeysDisabled": "Hotkeys disabled!",
     "hotkeyToggle": "Enable hotkeys",
-    "cheatSheet": "Hotkey Cheat Sheet"
+    "cheatSheet": "Hotkey Cheat Sheet",
+    "toggleColumnVisibility": {
+      "hide": "Columns that are hidden are now hidden to you",
+      "show": "Columns that are hidden are now visible to you"
+    },
+    "toggleShowOtherUsersNotes": {
+      "hide": "The notes of other participants are hidden",
```
Comment (Resaki1) [inline public/locales/en/translation.json:187]:
```suggestion
      "hide": "The notes of other participants are now hidden",
```

File: public/locales/en/translation.json [hunk 5]
```diff
@@ -178,7 +178,31 @@
...
+      "show": "The notes of other participants are now visible"
+    },
+    "toggleShowAuthors": {
+      "hide": "The authors of the notes are hidden",
```
Comment (Resaki1) [inline public/locales/en/translation.json:191]:
```suggestion
      "hide": "The authors of the notes are now hidden",
```

**Review by Resaki1 — CHANGES_REQUESTED** [review]
Idk if this is a new bug or not, but when multiple hotkeys are activated after another, the toasts are added to a queue. Which is fine, however every toast that is in the queue now appears two times:

https://user-images.githubusercontent.com/35272402/232825958-951c98b1-9ba5-4bd7-ba2b-f354aaca606e.mov

**Comment by brandstetterm** [comment]
> Idk if this is a new bug or not, but when multiple hotkeys are activated after another, the toasts are added to a queue. Which is fine, however every toast that is in the queue now appears two times:
> 
>  Bildschirmaufnahme.2023-04-18.um.17.28.43.mov

Thanks for the review. That's very strange behaviour and I'll try to fix it 👍🏼

**Review by Resaki1 — APPROVED** [review]

**Pull request merged**